### DISCUSSION TITLE
How to commit the code to the feature branch with a comment?

### DISCUSSION THREAD

**Discussion by maryamabasova** [LianqiangGuan/Tafe230211#9] [General]
Hello Team,

How to commit the code to the feature branch with a comment?

**Comment by LianqiangGuan** [comment]
Hi Mara
 When you use tortoiseGit to commit something, there will be a new window pop up, there is an area in this window to write your comment. I think the exercise session has some activities regards your question.
 Make sure you switch /checkout the branch that you would like to commit first, then do the commit
Hope this could help.

**Comment by billybena** [comment]
Could you also go git sync commit or do you just go git commit to the branch?
